checkov/cloudformation/checks/resource/aws/cloudwatchLogGroupRetention.py

```diff
@@ -0,0 +1,26 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.cloudformation.checks.resource.base_resource_check import BaseResourceCheck
+
+class cloudwatchLogGroupRetention(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure cloudwatch log groups specify retention days"
+        id = "CKV_AWS_66"
```

Comment:
What should we do with the 'id' field?  Wondering if this will conflict with the generation of the [resource-scans](https://github.com/bridgecrewio/checkov/blob/e5560e734f5328c2b9f8abd70e531dade21f20a0/docs/3.Scans/resource-scans.md) file